### PR TITLE
Test nameserver shutdown with a ConnectionClosedError

### LINKED ISSUE #279: Uncovered lines
This exception handling in `osbrain/proxy.py` is not always covered by the test suite:

```python
class Proxy():
    def shutdown(self, timeout=10.):
        # ...
        except ConnectionClosedError:
            pass
```

We should create a test that forces a `ConnnectionClosedError` on shutdown.

### PR DESCRIPTION
**Pull request by ocaballeror**
Fixes #279

### CODE REVIEW

File: osbrain/tests/test_proxy.py [hunk 1]
```diff
@@ -325,3 +329,13 @@ def shoot(self):
     assert not wayne._next_oneway
 
     assert wait_agent_attr(wayne, value=20 * ['bang!'], timeout=1.5)
+
+
+def test_agent_proxy_shutdown_connectionclosed():
```
Comment (Peque) [inline osbrain/tests/test_proxy.py]: `test_nameserver_proxy_*` instead?

Comment (Peque) [inline osbrain/tests/test_proxy.py]: Missing test docstring. Even if just something like "ConnectionClosedError exceptions are allowed when shutting down a name server from a proxy".

Comment (ocaballeror) [reply osbrain/tests/test_proxy.py]: Copy-pasting is a hard thing to do well...

Comment (Peque) [reply osbrain/tests/test_proxy.py]: Or even better: "... when remotely shutting down the name server daemon from a proxy".

Or whatever you come up with. Just something a bit descriptive. :joy:

Comment (Peque) [reply osbrain/tests/test_proxy.py]: I know how you feel. :joy:

**Pull request merged**